src/proto/FunctionRpc.proto

```diff
@@ -430,6 +430,7 @@ message TypedData {
     CollectionString collection_string = 9;
     CollectionDouble collection_double = 10;
     CollectionSInt64 collection_sint64 = 11;
+    BindingData bindingData = 12;
```

Comment:
If you write this like `binding_data` it will still be referenced w/ camel case in host/worker where they consume the protobuf file and it matches the official [style guide](https://developers.google.com/protocol-buffers/docs/style#message_and_field_names).

Reply:
done! updated all camelCase to be _